src/pages/Debug/DebugDetails.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useState} from 'react';
+import Button from '@components/Button';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import TextInput from '@components/TextInput';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import DebugUtils from '@libs/DebugUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import type {TranslationPaths} from '@src/languages/types';
+
+type DebugDetailsProps = {
+    data: Record<string, unknown>;
+    onSave: (data: Record<string, unknown>) => void;
+    onDelete: () => void;
+    validate: (key: never, value: string) => void;
+};
+
+function DebugDetails({data, onSave, onDelete, validate}: DebugDetailsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const [draftData, setDraftData] = useState<Record<string, string>>(DebugUtils.onyxDataToDraftData(data));
+    const [errors, setErrors] = useState<Record<string, string>>({});
+    return (
+        <ScrollView
+            style={styles.mt5}
+            contentContainerStyle={[styles.gap5, styles.ph5, styles.pb5]}
+        >
+            {Object.entries(draftData ?? {}).map(([key, value]) => (
```

Comment:
Currently, in our codebase, we use FormProvider in almost form, Could you refactor to use FormProvider here?

Reply:
NAB: we can leave it away